HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/PaymentComponent/PaymentComponentConfiguration.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  PaymentComponentConfiguration.swift
+//
+//  Copyright © 2024 Gini GmbH. All rights reserved.
+//
+
+
+import Foundation
+
+public struct PaymentComponentConfiguration {
+    /**
+      * Please contact a Gini representative before changing this configuration option.
+      */
+    public var isPaymentComponentBranded: Bool = true
```

Comment:
Yes. This configuration is optional. By default should always be true and in the future, we will probably add more configurations to this Payment Component so that this branded configuration shouldn't be mandatory to initialize. 
I am open to other suggestions.